recipe/meta.yaml

```
@@ -16,7 +16,7 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  script: TileDB_DIR=${PREFIX} {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```

```suggestion
  script: TILEDB_PATH=${PREFIX} {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```

From our discussion in https://github.com/TileDB-Inc/TileDB-Vector-Search/pull/361, I believe this should now be `TILEDB_PATH=${PREFIX}`